src/compressed_tensors/quantization/quant_args.py

```diff
@@ -94,7 +95,7 @@ class QuantizationArgs(BaseModel, use_enum_values=True):
     block_structure: Optional[str] = None
     dynamic: bool = False
     actorder: Union[ActivationOrdering, bool, None] = None
-    observer: str = Field(
+    observer: Optional[str] = Field(
         default="minmax",
```

Comment:
```suggestion
        default=None,
```

Reply:
You can do this or you can set observer=None for all dynamic schemes so they don't throw a warning

Reply:
I think we still want to keep the default to minmax since that will be the observer for majority of our cases.